### DISCUSSION TITLE
support for rog azoth keyboard?

### DISCUSSION THREAD

**Discussion by scr0-0ge** [seerge/g-helper#2208] [Ideas]
Just bought ROG Azoth (keyboard) and ROG Harpe Ace Aim Lab Edition (mouse)  today, I wonder if there is any plans to support this keyboard in the future?

Wish list: 
1.  Due to both of the device uses rog omni receiver, I would like to see the ability to pairing them together with one receiver like [ armory crate ](https://www.asus.com/support/faq/1050878/) does.
2. Ability to customize the oled screen on the Azoth keyboard, apparently oled burn in on this screen is a thing, I hope I could turn this screen off if necessary

**Comment by nistaani** [comment]
This would be amazing. G-Helper is so good I'd forgotten already about armory crate. But had to install it again for azoth options. 

Any way to help make this happen?

**Comment by IceStormNG** [comment]
Problem is that I need one of those devices here to test it and I can only abuse Amazon's return policy for so much that they don't ban me.

Theorefore I'm currently focusing on mice as I already did the big initial analysis of the packet flow.

I also still need to figure out how to detect which device is which behind the omni receiver. Right now it assumes an omni receiver represents the harpe ace. But more and more mice come with an omni receiver as it seems.

**Reply by CazyDave** [reply]
if you live anywhere near NJ id be happy to lend you my Azoth.

**Reply by IceStormNG** [reply]
Thanks for the offer, though, I live in Germany. I'm pretty sure that you don't wanna ship it around the world 😄
